CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - The `try_fold` function in the `iterator` module is now tail recursive.
 - The performance of many functions in the `string` module on the JavaScript
   target has been improved.
+- The `string.repeat` function has been optimised.
```

Comment:
This seems a bit redundant with the entry above?

Reply:
Collapsed to a single and more generic changelog entry